client/src/components/Navbar/index.tsx

```diff
@@ -13,11 +13,12 @@ const Navbar = (): ReactElement => {
 
   return (
     <Menu attached="top" className="navbar" borderless size="large">
-      <Menu.Item className="header" as={isOnboarded && Link} to="/homepage">
+      <Menu.Item className="header" as={isOnboarded && Link} to="/home">
```

Comment:
We forgot to say this. But the South Side Weekly header in the navbar shouldn't be clickable anymore so you can remove this link.